packages/transit-vehicle-overlay/src/TransitIcons.tsx

```diff
@@ -4,11 +4,20 @@ import React from "react";
 import { Bus, Streetcar, Ferry } from "@opentripplanner/icons";
 import styled, { css } from "styled-components";
 
-const rounded = css<{ rotate?: number; routeColor?: string }>`
-  background: #eee;
+const getBackgroundColor = routeColor => routeColor || "#9999ee";
+
+const rounded = css<{
+  disableHoverEffects?: boolean;
+  rotate?: number;
+  routeColor?: string;
+}>`
+  background: ${props =>
+    props.disableHoverEffects
+      ? getBackgroundColor(props.routeColor)
+      : "#eeeeee"}aa;
```

Comment:
Nitpick: What does transparency provide? Can we remove transparency? Or if the color needs to be "lightened up", could we do that instead?

Reply:
Transparency looks good. Solid blocks can be jarring.